README.md

```diff
@@ -61,19 +59,45 @@ ssb-server log
 # stream all messages by one feed, ordered by sequence number
 ssb-server hist --id $FEED_ID
 ```
+## Example Usage (js)
+
 ```js
-// In javascript:
+var Server = require('ssb-server')
+var config = require('ssb-config')
+var fs = require('fs')
+var path = require('path')
+
```

Comment:
Just tried to run this example and it threw an error about my lack of keys. I had to add this:

```js
config.keys = require('ssb-keys').loadOrCreateSync(path.join(config.path, 'secret'))
```

Reply:
right, nice catch! (I wish JS had that rust thing where example code gets run as a real test!)

I have personal feeling that should be in `ssb-config` yeah?
As in, you have to do this _every time_ right, and that things job is to get the default shit together for the server yeah?

Reply:
I hadn't really considered that, but yeah, that sounds exactly like something that should be declared in ssb-config.

Reply:
here's the PR : https://github.com/ssbc/ssb-config/pull/29